crates/swc_ecma_parser/src/lexer/number.rs

```diff
@@ -274,13 +313,17 @@ impl<'a, I: Input> Lexer<'a, I> {
         }
 
         let raw_str = raw.0.take().unwrap();
+        // Remove number separator from number
+        let raw_number_str = raw_str.replace('_', "");
```

Comment:
There is another solution - create the second buffer and store number without number separator and with, but don't think it will be effective here